moduleframework/dockerlinter.py

```diff
@@ -161,11 +159,15 @@ def get_specific_label(self, label_name=None):
         if label_name is None:
             return []
         label_list = self.get_docker_labels()
-        return [label_name in label_list]
+        return [label_list[key] for key in label_list.keys() if label_name == key]
 
-    def check_baseruntime(self):
+    def check_from_is_first(self):
```

Comment:
Docs of this method and its implementation are two completely different things.

Reply:
Fixed.